src/components/recovery/GroupedRecoveryListItems/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box, Paper, Typography } from '@mui/material'
+import { partition } from 'lodash'
+import type { RecoveryQueueItem } from '@/services/recovery/recovery-state'
+import type { Transaction } from '@safe-global/safe-gateway-typescript-sdk'
+import type { ReactElement } from 'react'
+
+import { isRecoveryQueueItem } from '@/utils/transaction-guards'
+import ExpandableTransactionItem from '@/components/transactions/TxListItem/ExpandableTransactionItem'
+import { RecoveryListItem } from '../RecoveryListItem'
+import ExternalLink from '@/components/common/ExternalLink'
+
+import css from '@/components/transactions/GroupedTxListItems/styles.module.css'
+
+function Disclaimer({ isMalicious }: { isMalicious: boolean }): ReactElement {
+  return (
+    <Box
+      className={css.disclaimerContainer}
+      sx={{ bgcolor: ({ palette }) => `${palette.warning.background} !important` }}
+    >
+      <Typography>
+        <Typography component="span" fontWeight={700}>
+          Rejecting {isMalicious ? 'malicious transaction' : 'Account recovery'}.
+        </Typography>{' '}
+        You can approve or reject the cancellation.{' '}
```

Comment:
This wording is a bit confusing.

> Rejecting Account recovery. You can approve or reject the cancellation.

We should call it either rejection or cancellation, not both in one centence. On top of that, rejecting a cancellation is even more confusing.

Reply:
I've renamed it the following:

> Cancelling the Account recovery. You will need to execute the cancellation.